README.md

```diff
@@ -27,6 +31,10 @@ In order to run GraphSpace, please install postgreSQL and both the Python runtim
 3. Create a virtual environment for the project: `virtualenv venv`
 4. Start using the virtual environment: `source venv/bin/activate`
 5. Install graphspace: `sh install.sh`
+   
+   _Note : While installing psycopg2, you might encounter 
+   [```Error: You need to install postgresql-server-dev-X.Y for building a server-side extension or libpq-dev for building a client-side application.```](https://stackoverflow.com/q/28253681/4646197)
+   To fix it follow the solution given [here](https://stackoverflow.com/a/28254860/4646197)._
```

Comment:
I feel like this is a more common error than 10%: you would need to explicitly have installed these dev libraries for postgres in the past.If you don't get this error, then you haven't installed it.

Maybe we should list it as a separate dependency, perhaps as a sub-bullet under Postgres?